src/test/java/org/apache/commons/io/FileUtilsTestCase.java

```diff
@@ -1015,6 +1021,12 @@ public void testIsFileNewerOlder() throws Exception {
 
         final Date date = new Date();
```

Comment:
In general, you want to use hard coded dates in order to avoid tests failing on the two times a year the build runs in a locale that happens to perform a daylight-saving-time switch. Yes, I've actually seen this happen (at work.)

Reply:
> 
> 
> In general, you want to use hard coded dates in tests

The files being tested seem to be created at the moment the test is run, so how should it be handled?

> Yes, I've actually seen this happen (at work.)

Wow.